### ISSUE TITLE
New Chrome "select next tab" shortcut conflicts

### ISSUE THREAD

**Issue by wkoffel** [eczarny/spectacle#603]
I blame Chrome for this, but curious if anyone has found a workaround so that Spectacle takes precedence over local apps.  I woke up today to find that a new version of Chrome (`Version 50.0.2661.102 (64-bit)`) has hijacked my option-cmd-arrow keys to switch tabs.  I use command+[] to change tabs, but I can't figure out how to disable this new default in Chrome.

Anyone run into this as well?

**Comment by wkoffel** [comment]
Nevermind.  I just went into my mail program, and found a similar conflict.  I then closed and relaunched Spectacle, and things are working again.  Here's what I suspect happened.

I updated Spectacle, and when it relaunched, it somehow didn't regrab global control.  Or maybe the launch order of apps upon a reboot caused it.  But for posterity, I'll leave this open and suggest: **quit and relaunch Spectacle**
